src/Illuminate/Mail/Mailer.php

```diff
@@ -382,7 +384,22 @@ protected function sendSwiftMessage($message)
         }
 
         try {
-            return $this->swift->send($message, $this->failedRecipients);
+            $result = $this->swift->send($message, $this->failedRecipients);
```

Comment:
👎  on this PR purely due to this. We intentionally hid this because this is meant to be an abstraction.